BrightID/src/utils/operations.js

```diff
@@ -5,14 +5,20 @@ import fetchUserInfo from '../actions/fetchUserInfo';
 import api from '../Api/BrightId';
 
 export const pollOperations = async () => {
+  const time_fudge = 2 * 60 * 1000; // trace operations for 2 minutes
   try {
     const { operations } = store.getState();
     for (const op of operations) {
       const { state } = await api.getOperationState(op);
-      console.log('state', state);
-      if (state === 'applied') {
+      if (state === 'applied' || state === 'failed') {
         store.dispatch(removeOperation(op));
         store.dispatch(fetchUserInfo());
+        continue;
```

Comment:
the only thing left to do is the same as what happens if the state is applied or failed